src/screens/Order/HSwitchOrderUtils.res

```diff
@@ -224,3 +224,47 @@ module EllipsisText = {
     </div>
   }
 }
+let validateIsNull = value => {
+  switch value {
```

Comment:
its only has Some(_) or None , no need of checking for _

Reply:
in case of none or some(val) where val is nulljson I am returning true  and in any case where I can get the val without any of above case it will return false